src/assets/resources.json

```
@@ -748,6 +748,18 @@
         "profile": "https://github.com/bradtaniguchi"
       }
     },
+    {
+  "title": "angular-electron",
+  "description": "useful resource for cross platform development",
+  "author": {
+    "name": "Maxim Gris",
+    "profile": "A link to the author's bio - this is optional"
+  },
+  "url": "https://github.com/maximegris/angular-electron",
+  "category": "library",
+  "contributor": {
+    "name": "Phil", }
```

This has to be a valid JSON format, could you also fix the indentation please?

```suggestion
    "name": "Phil" }
```